.circleci/config.yml

```diff
@@ -239,9 +255,6 @@ jobs:
       python_version: <<parameters.python_version>>
     steps:
       - setup
-      - run:
-          name: Install pre-commit hooks
-          command: pre-commit install --install-hooks
```

Comment:
Turns out this isn't needed - `make lint` will install them as needed. It wasn't there on the `private-kedro-sync` job that did linting and that all worked fine.